src/extensions/twowaycontactsyncadapter_impl.h

```diff
@@ -313,11 +365,17 @@ bool detailPairExactlyMatches(const QContactDetail &a, const QContactDetail &b)
                 // this is ok.
             } else {
                 // a has a real value which b does not have.
+                if (Q_UNLIKELY(printDifferences)) {
```

Comment:
I bet you can't possibly create a microbenchmark using QDebug that shows any measurable benefit of using Q_UNLIKELY...